server-side-rendering/src/routes/+layout.server.ts

```diff
@@ -0,0 +1,7 @@
+export function load({ locals }) {
```

Comment:
Is this typed automatically? If not, lets type those.

Reply:
It's typed automatically.